spec/features/user_can_see_login_form_spec.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
```

Comment:
This feature file only checks for items, but no functionality, and is in my opinion not necessary as we check the functionality and the existence with the `login` feature